dev/com.ibm.ws.microprofile.config.1.1.cdi/resources/com/ibm/ws/microprofile/config/cdi/resources/ConfigCDI.nlsprops

```diff
@@ -48,6 +48,11 @@ property.not.found.CWMCG5004E=CWMCG5004E: The "{0}" configuration property was n
 property.not.found.CWMCG5004E.explanation=The named property was not found in any of the configured sources. 
 property.not.found.CWMCG5004E.useraction=Ensure that the property exists in one of the Config Sources.
 
+The InjectionPoint dependency was not resolved for the Observer method: {0}. Error: {1}.
+unable.to.resolve.observer.injection.point.CWMCG5005E=CWMCG5005E: The InjectionPoint dependency was not resolved for the Observer method: {0}. Error: {1}.
+unable.to.resolve.observer.injection.point.CWMCG5005E.explanation=An injected Config Property did not resolve. Either the named property could not be found or a Converter for the required type did not resolve. 
+unable.to.resolve.observer.injection.point.CWMCG5005E.useraction=Check the server message and ffdc logs to determine the cause. Ensure that the property exists in one of the Config Sources and that the required Converters have been configured.
```

Comment:
Change to
"Check the server message and FFDC logs to determine the cause. Ensure that the property exists in one of the ConfigSources and that the required Converters were configured."

Reply:
done